### DISCUSSION TITLE
Welcome to conv-emotion Discussions!

### DISCUSSION THREAD

**Discussion by soujanyaporia** [declare-lab/conv-emotion#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Coding511** [comment]
In this dilogueGCN model on IEMOCAP using your model, I am getting 55% F1 score.


epoch: 60, train_loss: 0.588, train_acc: 80.64, train_fscore: 80.52,
                 valid_loss: nan, valid_acc: nan, valid_fscore: nan, 
                  test_loss: 1.0801, test_acc: 54.65, test_fscore: 54.73, 
                 time: 3.26 sec
                    Test performance..
                  F-Score: 55.0


In your paper, you are claiming 65.25 
please explain

**Reply by deepanwayx** [reply]
Can you share the full command that you are using for training?

You can also use the alternate implementation in the `DialogueGCN-mianzhang` directory.

**Comment by Coding511** [comment]
Sir
Downloaded the code, along with the IEMOCAP features provided.
Just run the train.py file.
 and after 60 Epochs :

epoch: 60, train_loss: 0.5595, train_acc: 82.86, train_fscore: 82.76,
*valid_loss: nan, valid_acc: nan, valid_fscore: nan, *
test_loss: 1.0062, test_acc: 61.49, test_fscore: 60.7,
time: 2.97 sec
Test performance..
F-Score: 60.79

Why valid data is nan, is it because nothing is split into this data
partition?
changing the seed giving me this accuracy and its changing somewhat
everytime.

Is there anything i need to tune for your accuracy sir?

Regards

**Comment by Yogurt928** [comment]
Hi Sir,

What's the difference between the two versions of implementation of DialogueGCN?

Thanks.
